kubernetes-crds/config/function-invoker-resource.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: functioninvokers.projectriff.io
+versions:
+- name: v1
+spec:
+  scope: Cluster
+  group: projectriff.io
+  version: v1
+  names:
+    kind: FunctionInvoker
```

Comment:
Since we are already within the scope of the riff namespace, I wonder if "invoker" would be sufficient. It would make things like completion much nicer than if we have 2 of our 3 resources beginning with "function".

Reply:
In the initial proposal the type was `Invoker` but I changed it to `FunctionInvoker` since all of our repos are `*-function-invoker`. But, I'd be happy to switch back.

Reply:
Ae they function-invokers for java/node/go/... or are they invokers for java/node/go/... functions? I think the latter sounds better so "Invoker" would be fine.

Reply:
back to Invoker